metadata.yml

```diff
@@ -0,0 +1,29 @@
+rules:
+  - apiGroups:
+      - "*"
+    apiVersions:
+      - "*"
+    resources:
+      - "*"
+    operations:
+      - CREATE
+      - UPDATE
```

Comment:
Using `*` is a problem when used with [audit-scanner](https://github.com/kubewarden/audit-scanner/blob/ad679a7eb527492dc9e3d01c462c5db355332794/internal/policies/client.go#L445-L457). Maybe let's keep inspecting the main workload resources. And, a comment that the policy can be used for all the resources types with annotations.